lang/src/account.rs

```diff
@@ -34,6 +37,9 @@ impl<'a, T: AccountSerialize + AccountDeserialize + Owner + Clone> Account<'a, T
     /// possible.
     #[inline(never)]
     pub fn try_from_unchecked(info: &AccountInfo<'a>) -> Result<Account<'a, T>, ProgramError> {
+        if system_program::ID.eq(info.owner) && info.lamports() == 0 {
```

Comment:
Why `.eq` instead of `==`?

Reply:
Between the use of `ref == &something` and `ref.eq(something)`, I prefer the latter, but I will correct but more accepted in the project (judging by the `&T::Owner()`)